content/en/docs/setup/additional-setup/external-istiod/index.md

```diff
@@ -0,0 +1,309 @@
+
+## Requirements
+
+### Clusters
+
+This guide requires that you have two Kubernetes clusters with any of the
+supported Kubernetes versions: {{< supported_kubernetes_versions >}}.
+
+The first cluster is the {{< gloss >}}external control plane{{< /gloss >}} cluster with the Istio `default` profile installed in the `external-istiod` namespace. This istiod serves as the the external control plane for the second cluster.  An ingress gateway is installed in the `istio-system` namespace. The ingress gateway provides mesh sidecars access to the external istiod in the `external-istiod` namespace.
+
+The second cluster is a {{< gloss >}}remote cluster{{< /gloss >}} hosting the mesh. Its Kubernetes API server also provides the configuration for the control plane (istiod) running in the external cluster.
+
+### API Server Access
+
+The API Server in the remote config cluster must be accessible to the external control plane cluster. Many cloud providers make API Servers publicly accessible via network
+load balancers (NLBs). If the API Server is not directly accessible, you will
+have to modify the installation procedure to enable access. For example, the
+[east-west](https://en.wikipedia.org/wiki/East-west_traffic) gateway used in
+the multi-network and primary-remote configurations could also be used
+to enable access to the API Server.
+
+## Environment Variables
+
+This guide will refer to two clusters named `external_cluster` and `remote_cluster`. The following environment variables will be used throughout to simplify the instructions:
+
+Variable | Description
+-------- | -----------
+`CTX_EXTERNAL_CLUSTER` | The context name in the default [Kubernetes configuration file](https://kubernetes.io/docs/tasks/access-application-cluster/configure-access-multiple-clusters/) used for accessing the external control plane cluster.
+`CTX_REMOTE_CLUSTER` | The context name in the default [Kubernetes configuration file](https://kubernetes.io/docs/tasks/access-application-cluster/configure-access-multiple-clusters/) used for accessing the remote config cluster.
+`EXTERNAL_ISTIOD_ADDR` | The external host name for the `remote_cluster` to access the external istiod.
+`SSL_SECRET_NAME` | The secret name used to access the ingress gateway on the external control plane cluster.
+
+For example:
+
+{{< text bash >}}
+$ export CTX_EXTERNAL_CLUSTER=external_cluster
+$ export CTX_REMOTE_CLUSTER=remote_cluster
+$ export REMOTE_ISTIOD_ADDR=myexternal-istiod.cloud.com
+$ export SSL_SECRET_NAME = myexternal-istiod-secret
+{{< /text >}}
+
+## Setup
+
+### Setup the external control plane cluster
+
+Create the Istio configuration for `external_cluster`, using the default profile with the following ports on the ingress gateway to expose the external istiod:
+
+{{< text bash >}}
+$ cat <<EOF > external-cp.yaml
```

Comment:
Should we use ingress-gateway or east-west gateway?

@stevenctl @costinm any thoughts?

Reply:
I would say eastwest – the whole idea was that we'd have different gateways to allow hardening and scaling the gateway used for non-ingress purposes separately from how we scale the gateway responsible for external traffic coming in.

Reply:
Note: this is the istio + ingress gateway that istiod is exposed on.  It is not just the gateway.  And this is slight different than the east-west gw we used in multicluster to allow communications among service a in cluster 1 to reach to service b in cluster 2. It is for sidecar in remote cluster to reach to external istiod.

Reply:
We use eastwest to reach istiod in the primary remote case. Shouldn't this be similar?

Reply:
I suppose neither `ingress-gateway` nor `eastwest-gateway` are very good names for this one. Maybe just call it `controlplane-gateway`?

Reply:
Thanks Frank, updated!